CONTRIBUTING.md

```diff
@@ -56,33 +56,7 @@ Terraform has similar standards, and you can run `terraform fmt` to rewrite Terr
 
 ## Commit Message Format
 
-We follow a rough convention for commit messages that is designed to answer two
-questions: what changed and why. The subject line should feature the what and
-the body of the commit should describe the why.
-
-```
-scripts: add the test-cluster command
-
-this uses tmux to set up a test cluster that you can easily kill and
-start for debugging.
-
-Fixes #38
-```
-
-The format can be described more formally as follows:
-
-```
-<subsystem>: <what changed>
-<BLANK LINE>
-<why this change was made>
-<BLANK LINE>
-<footer>
-```
-
-The first line is the subject and should be no longer than 70 characters, the
-second line is always blank, and other lines should be wrapped at 80 characters.
```

Comment:
Subject body is discussed in [`git-commit(1)`][1] and by Chris [here][2] and [here][3] (although both of those have an overly-strict 50-char subject guideline; I think our 70 is more acheivable).

[1]: https://git-scm.com/docs/git-commit#_discussion
[2]: https://chris.beams.io/posts/git-commit/#separate
[3]: https://chris.beams.io/posts/git-commit/#limit-50